packages/core/email/package.json

```diff
@@ -26,13 +26,22 @@
     "test:front:watch:ce": "cross-env IS_EE=false jest --config ./jest.config.front.js --watchAll"
   },
   "dependencies": {
+    "@strapi/design-system": "1.6.3",
+    "@strapi/icons": "1.6.3",
     "@strapi/provider-email-sendmail": "4.6.1",
     "@strapi/utils": "4.6.1",
-    "lodash": "4.17.21"
+    "axios": "1.2.2",
```

Comment:
Do we still need that now that @simotae14 moved it to use `useFetchClient`?

Reply:
If the helper-plugin does not require a peerDep of axios, we can remove it.

Reply:
🤔 Mh that's an interesting question actually: should we bundle axios (only) with the HP at some point? Right now you might right and it is a peer dep.

Reply:
I've checked. `axios` is a dependency of the helper-plugin and not listed as peer-dep.

Reply:
When this list was made, these files depended on it, and now I believe we can safely remove it with https://github.com/strapi/strapi/pull/15730!
https://gist.github.com/smorimoto/07f7df255735bb250d3116565261f24d#file-unsafe-packages-json-L740